CHANGELOG.md

```diff
@@ -1,10 +1,22 @@
 # Changelog
 
+# 4.0.0 - [#22](https://github.com/ServiceInnovationLab/openfisca-aotearoa/pull/22)
```

Comment:
> Perhaps surprisingly, if all the changes are additions to the model and a reorganisation of the files hierarchy, that means there were no changes to the public API of the model, and a minor version bump could be sufficient 🙂 ([semver cheat sheet](https://speakerdeck.com/mattisg/git-session-2-strategies?slide=81))

Reply:
I agree, but I've merged the previous rates_rebates pr which required a major release into this one.